app/helpers/application_helper.rb

```diff
@@ -76,8 +76,12 @@ def icon_url(name)
     "https://res.cloudinary.com/#{ApplicationConfig['CLOUDINARY_CLOUD_NAME']}/image/upload/#{postfix}"
   end
 
+  def idn_domain_fix(url)
```

Comment:
**thought (non-blocking):** This helper doesn't seem to serve much of a purpose. Maybe for now we can just inline the call to `SimpleIDN` in `cloudinary` and update the test to check the URL in the attribute. I'll accept the PR either way, but if you do change it please request a re-review.

Reply:
That's a good point, @citizen428. My first inclination is to break things out in their own methods for reuse, but you're right, here there isn't much of a reuse potential. I can take another crack at this and revise the specs this evening my time and I'll re-request a review then.